ddl/disttask_flow.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 PingCAP, Inc.
```

Comment:
Maybe we can put these files to `disttask/ddl`?

Reply:
I think it's better to put it in `ddl` package now, the `litBackfillFlowHandle` is referring a private def `ddl` object. Event though only `ddl.store` are used but I'm not sure whether it will use other fields in the later dev progress...

Reply:
DDL package is too big. if you change anything in ddl, all cache of ddl will be unavailable. so Can you create ```dddl/disktask``` ?